File: web/palindrome.php
<?php

function palindromeChecker($n) {
    $rev = 0.0;
    $rem = 0.0;
    $init = $n;

    for ($rev = 0; $n >= 1;) {
	$rev *= 10;
	$rem = $n % 10;
	$rev += $rem;
	$n /= 10;
    }

    echo "$rev\n";

    if ($rev == $init) echo "$init is a palindrome.";
    else echo "$init is not a palindrome.";
}

$num = 69;
palindromeChecker($num);

?>

File: c/palindrome.c
// WAP to find if given number is palindrome or not
#include <stdio.h>

int main(void) {
	int num, num_used, rev = 0, rem;

	printf("Enter a number\n");
	scanf("%d", &num);
	num_used = num;

	while (num_used != 0) {
		rem = num_used % 10;
		rev = ( rev * 10 ) + rem;
		num_used /= 10;
	}

	printf("The number is %s", rev == num ? "a palindrome" : "not a palindrome");

	return 0;
}
